client/src/api/httpClient.js

```diff
@@ -0,0 +1,33 @@
+import axios from 'axios';
+
+const baseURLs = {
+	production: '',
+	development: 'http://localhost:3001/'
+};
+const baseURL = baseURLs[process.env.NODE_ENV];
```

Comment:
we shouldn't have to do this because we have a proxy set up - you can try it and see that it hits the back-end